packages/react-core/src/components/Label/Label.tsx

```diff
@@ -183,17 +183,19 @@ export const Label: React.FunctionComponent<LabelProps> = ({
   const LabelComponent = (isOverflowLabel ? 'button' : 'span') as any;
 
   const button = closeBtn ? (
-    closeBtn
+    <span className={css(styles.labelActions)}>closeBtn</span>
```

Comment:
```suggestion
    <span className={css(styles.labelActions)}>{closeBtn}</span>
```

Reply:
whoops - couldn't find any example of this being used, do we know if anyone using a custom close button?

Reply:
Yeah I had to pass the closeBtn prop in manually to test it. Only use case I can think of is something along the lines of wanting to use a trash can icon instead of the `x`

Reply:
FYI per request from @mcoker I checked the projects we currently track and found `closeBtnAriaLabel` and `closeBtnProps` but no use of just `closeBtn`.  Here is the list of files with the props mentioned above:

```
- Migration-Toolkit-for-Containers/src/app/home/pages/PlansPage/components/Wizard/MigrationOptions/MigrationOptionsForm.tsx
- Kogito/packages/pmml-editor/src/editor/components/DataDictionary/PropertiesLabels/PropertiesLabels.tsx
- Kogito/packages/pmml-editor/src/editor/components/MiningSchema/MiningSchemaFieldLabels/MiningSchemaFieldLabels.tsx
- Infinispan/src/app/Caches/Create/Features/IndexedCacheConfigurator.tsx
- Cockpit-machines/src/components/common/stateIcon.jsx
```